src/Form.vue

```diff
@@ -346,6 +353,20 @@ export default {
               }
             }
 
+            // evaluate default()
```

Comment:
Reversing the order of evaluating `default()` and `choices()` could cause issues:
For example, you can tell that a default value of 10 is invalid if there are only 9 items in the list; therefore, you must evaluate the choices before the evaluating the default value.

Reply:
I couldn't find in inquirer.js documentation  that they mention the functions evaluate order.

![image](https://user-images.githubusercontent.com/57438361/85366567-48a43480-b530-11ea-9d9c-8f2a4e9ab46d.png)

Reply:
done